packaging/MacOs/build_macos.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2023 UltiMaker
+# Cura is released under the terms of the LGPLv3 or higher.
+
+
+import os
+import argparse  # Command line arguments parsing and help.
+import subprocess
+from pathlib import Path
+
+ULTIMAKER_CURA_DOMAIN = os.environ.get("ULTIMAKER_CURA_DOMAIN", "nl.ultimaker.cura")
+
+def build_pkg(dist_path: str, app_filename: str, component_filename: str, installer_filename: str) -> None:
+    """ Builds and signs the pkg installer.
+
+    @param dist_path: Path to put output pkg in
+    @param app_filename: name of the .app file to bundle inside the pkg
+    @param component_filename: Name of the pkg component package to bundle the app in
+    @param installer_filename: Name of the installer that contains the component package
+    """
+    pkg_build_executable = os.environ.get("PKG_BUILD_EXECUTABLE", "pkgbuild")
+    product_build_executable = os.environ.get("PRODUCT_BUILD_EXECUTABLE", "productbuild")
+    codesign_identity = os.environ.get("CODESIGN_IDENTITY")
```

Comment:
Default to None such that community member can build locally without the signing
```suggestion
    codesign_identity = os.environ.get("CODESIGN_IDENTITY", None)
```

Reply:
The default return for this in None https://docs.python.org/3/library/stdtypes.html#dict.get